lib/rubocop/cop/rspec/shared_examples.rb

```diff
@@ -72,6 +92,37 @@ def offense?(ast_node)
           end
         end
 
+        def register_method_name_offense(node)
+          preferred_method_name = preferred_method_name(node.method_name)
+          return unless preferred_method_name
+          return if node.method?(preferred_method_name)
+
+          selector = node.loc.selector
+          message = format(
+            MSG_INCORRECT_METHOD,
+            prefer: preferred_method_name,
+            current: node.method_name
+          )
+
+          add_offense(selector, message: message) do |corrector|
+            corrector.replace(selector, preferred_method_name)
+          end
+        end
+
+        def preferred_method_name(method_name)
+          if SharedGroups.examples(method_name)
+            cop_config['PreferredExamplesMethod']
+          elsif SharedGroups.context(method_name)
+            cop_config['PreferredContextMethod']
+          elsif Includes.examples(method_name)
+            cop_config['PreferredIncludeExamplesMethod']
+          elsif Includes.context(method_name)
+            # :nocov: - simplecov is not detecting the coverage of this branch!
+            cop_config['PreferredIncludeContextMethod']
+            # :nocov:
+          end
```

Comment:
I'm unclear as to why, but simplecov doesn't think this branch is covered, even though it is (line 270 of the spec).